combat-trainer.lic

```diff
@@ -3136,6 +3136,7 @@ class GameState
     echo("  @ambush: #{@ambush}") if $debug_mode_ct
 
     @backstab = settings.backstab
+    @backstab_past_mindlock = settings.backstab_past_mindlock
```

Comment:
Please add a default value to `base.yaml`

Reply:
Sorry this took so long, added it.